### ISSUE TITLE
Problem with Opera

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/scribefire-chrome#567]
```
What's the problem?

I can't login my blog to scribdfire



What browser are you using?

opera



What version of ScribeFire are you running?
11.61
```

-----
Original issue reported on code.google.com by `ltsaou...@gmail.com` on 4 Feb 2012 at 3:13

**Comment by GoogleCodeExporter** [comment]
```
which blog type are you using and if possible download 4.3.1 and try again it 
may solve your issue
```

-----
Original comment by `scribefi...@gmail.com` on 14 Apr 2014 at 7:23
* Changed state: **Pending**

**Comment by GoogleCodeExporter** [comment]
```
fixed in the version 4.3.1
```

-----
Original comment by `scribefi...@gmail.com` on 10 Jun 2014 at 8:01
* Changed state: **Fixed**
